api-guidelines/async/semantics/event-structure/rules/should-send-durable-event-ids.md

```diff
@@ -0,0 +1,9 @@
+---
+id: R000058
+---
+
+# SHOULD send durable event IDs 
```

Comment:
I'm not into this from a technical perspective. But when reading this headline and then the text below, to me this somehow doesn't fit together.

Reply:
we agreed to leave it for now as is